### PR TITLE
[feat]tools-v2: add bs check consistency

### PR DESCRIPTION
**Pull request by victorseptember**
### What problem does this PR solve?

Issue Number: #2351  

Problem Summary: Add bs check consistency command.

### What is changed and how it works?

What's Changed:
modify tools-v2/README.md
modify tools-v2/internal/error/error.go
modify tools-v2/pkg/cli/command/curvebs/check/check.go
modify tools-v2/pkg/cli/command/curvebs/query/seginfo/seginfo.go
modify tools-v2/pkg/cli/command/curvebs/status/copyset/agent.go
add tools-v2/pkg/cli/command/curvebs/check/consistency/consistency.go

How it Works:
Usage:
```shell
curve bs check consistency --path /test
```
Output:
```shell
+--------+--------+------------+
|  NAME  | STATUS |   EXPLAIN  |
+--------+--------+------------+
| /test  |  ok    |      -     |
+--------+--------+------------+
```
Side effects(Breaking backward compatibility? Performance regression?):

### Check List

- [x] Relevant documentation/comments is changed or added
- [x] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

**Comment by wuhongsong** [comment]
cicheck

**Comment by victorseptember** [comment]
cicheck

**Comment by Cyber-SiKu** [comment]
cicheck

**Comment by caoxianfei1** [comment]
cicheck

File: tools-v2/pkg/cli/command/curvebs/check/consistency/consistency.go [hunk 1]
```diff
@@ -0,0 +1,194 @@
+/*
+ *  Copyright (c) 2022 NetEase Inc.
```
Comment (Cyber-SiKu) [inline tools-v2/pkg/cli/command/curvebs/check/consistency/consistency.go:2]:
```suggestion
 *  Copyright (c) 2023 NetEase Inc.
```

**Comment by Cyber-SiKu** [comment]
cicheck

**Comment by Cyber-SiKu** [comment]
cicheck

**Comment by victorseptember** [comment]
cicheck

**Comment by Cyber-SiKu** [comment]
cicheck

**Comment by Cyber-SiKu** [comment]
cicheck

**Review by Cyber-SiKu — APPROVED** [review]

**Review by caoxianfei1 — APPROVED** [review]

**Comment by Cyber-SiKu** [comment]
cicheck

**Pull request merged**